ssh-utils/ssh-pubkey.ps1

```diff
@@ -5,14 +5,14 @@
 
 if (!(Test-Path -Path "$Env:USERPROFILE/.ssh"))
 {
```

Comment:
Sorry, this line SHOULD get the -Force. It IS a directory.

```
New-Item -Path "$Env:USERPROFILE/.ssh" -ItemType Directory -Force
```

Github attached my comment to the wrong line.

Reply:
@coolaj86... hope its all done now